rms/file_processing/managers.py

```diff
@@ -0,0 +1,16 @@
+from rms.file_processing.models import File
+from rms.utils.managers import BaseModelManager
+from sqlalchemy.orm import Session
+from rms.utils.postgres import engine
+
+
+class FileManager(BaseModelManager[File]):
+    __model__ = File
+
+    @classmethod
+    def create(cls, **kwargs) -> File:
+        with Session(engine) as session:
+            file_instance = cls.__model__(**kwargs)
+            session.add(file_instance)
+            session.commit()
+            return file_instance
```

Comment:
This is very task specific function.
Imo a better signature for this function would be

```python
@classmethod
def create(cls, instance: File) -> File:
    pass
```

So you have to create the instance before passing it to the manager to be saved to the database

And an even better solution would be to put this inside the `BaseModelManager` and change the `File` type to the generic `T` type

Reply:
fixed